### PR TITLE
[RESTEASY-2738]:Create bootable jar example

### PR DESCRIPTION
**Pull request by jimma**

### CODE REVIEW

**Review by liweinan — APPROVED** [review]

**Comment by liweinan** [comment]
@asoldano @ronsigal Please help to review this PR too and I'll merge it.

**Review by asoldano — APPROVED** [review]
Looks OK to me, thanks

File: bootable-jar/pom.xml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss.resteasy</groupId>
+        <artifactId>testable-examples-pom</artifactId>
+        <version>4.1.2.Final-SNAPSHOT</version>
```
Comment (rsearls) [inline bootable-jar/pom.xml:8]: Hmmmm,  version is "4.1.2.Final-SNAPSHOT".   Shouldn't it be either a  Final or a SNAMPSHOT?

File: bootable-jar/pom.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>products-runner</artifactId>
+    <packaging>war</packaging>
+    <name>Resteasy Bootable JAR Example</name>
+    <description>Reste bootable jaxrs jar Example</description>
```
Comment (rsearls) [inline bootable-jar/pom.xml:14]: typeo  "Reste"  should be "Rest".

**Review by rsearls — COMMENTED** [review]
Looks good.  Just minor changes.

**Pull request merged**